Samples/Samples/ViewModel/FilePickerViewModel.cs

```diff
@@ -136,12 +136,16 @@ async Task<FileResult> PickAndShow(PickOptions options)
 
                 if (result != null)
                 {
-                    Text = $"File Name: {result.FileName}";
+                    var size = await GetStreamSizeAsync(result);
```

Comment:
it seems inefficient to open the stream twice. First here to get the size.

then later (if an image) to read it's content.

Reply:
That is partially the point. I want to test a double open and close of the stream.

Reply:
yep. but it's not obvious (no comment) and sample code tends to be copy/pasted into prod apps
still meant it as something quite minor :)